Reading List/Reading List/BookDetailViewController.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  BookDetailViewController.swift
+//  Reading List
+//
+//  Created by James Sedlacek on 6/5/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import UIKit
+
+class BookDetailViewController: UIViewController {
+
+    var bookController: BookController?
+    var book: Book?
+    @IBOutlet weak var bookTitleTextField: UITextField!
+    @IBOutlet weak var reasonToReadTextView: UITextView!
+    
+    @IBAction func saveButtonTapped(_ sender: Any) {
+        // either call the createBook method in the bookController
+        // if the book property is nil
+        
+        if let theBook = book {
+            bookController?.updateBook(book: theBook,
+                                       newBookTitle: bookTitleTextField.text ?? nil,
+                                       newReasonToRead: reasonToReadTextView.text ?? nil)
+        } else {
+            // or call the update method in the bookController
+            // if the book property is not nil.
+            bookController?.createBook(title: bookTitleTextField.text ?? "", reasonToRead: reasonToReadTextView.text ?? "")
```

Comment:
It might be wise to check to make sure they entered a title as currently they could save an entry with no book title which ends up just appearing as a check box in the table view.